### PR TITLE
Add bug, feature request, and new check issue templates

### PR DESCRIPTION
**Pull request by danielduhh**
### Description:

This adds 3 new issue templates:
 - bug report
 - feature request
 - new check request

### Potential Impact:

N/A

### Unit Test Approach:

N/A

### Test Results:

N/A

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-integrity-check.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ - Summary/Explanation of what is being flagged and why
+ - OSM Link
+
+Add image here
+
+**Further investigation**
+Provide any additional information found that is worth mentioning (e.g. Further explanation of false positives or false negatives).
+
+**Supported regions**
+Specify which countries/regions this check is applicable to. List ISO codes. All countries in a valid answer.
```
Comment (sayas01) [inline .github/ISSUE_TEMPLATE/new-integrity-check.md:35]: typo: All countries ~in~ is a valid answer.

File: .github/ISSUE_TEMPLATE/new-integrity-check.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+1. Check Candidate (example)
+ - Is an `Edge`|`Point`|`Relation`|`Node`|`Area`|`Line`|
+ - Has `highway` tag, values include `trunk_link`, `trunk`, `motorway_link`
+
+**Use Cases**
+This section summarizes the various use cases for this check and provides potential enhancement suggestions if needed. The uses cases should act as a test in a way, such that when the analysis/enhancement is completed, these types of use cases should be found or eliminated.
+
+For each use case, please add a screenshot of the feature & openstreetmap.org link (or other sources). If necessary, edit the screenshot to better illustrate the use case.
+
+Example:
```
Comment (sayas01) [inline .github/ISSUE_TEMPLATE/new-integrity-check.md:24]: nit: Can you add a line break here?

File: .github/ISSUE_TEMPLATE/new-integrity-check.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+*Case 1: [Descriptive but concise title.] (Location)*
+ - Summary/Explanation of what is being flagged and why
+ - OSM Link
+
+Add image here
```
Comment (sayas01) [inline .github/ISSUE_TEMPLATE/new-integrity-check.md:29]: nit: Add _supporting_ image here

File: .github/ISSUE_TEMPLATE/new-integrity-check.md (hunk 2 above)
Comment (danielduhh) [inline .github/ISSUE_TEMPLATE/new-integrity-check.md:24]:
```suggestion
Example:

```

File: .github/ISSUE_TEMPLATE/new-integrity-check.md (hunk 3 above)
Comment (danielduhh) [inline .github/ISSUE_TEMPLATE/new-integrity-check.md:29]:
```suggestion
Add supporting  image here
```

File: .github/ISSUE_TEMPLATE/new-integrity-check.md (hunk 1 above)
Comment (danielduhh) [inline .github/ISSUE_TEMPLATE/new-integrity-check.md:35]:
```suggestion
Specify which countries/regions this check is applicable to. List ISO codes. "All countries" works all well.
```

**Pull request merged**